### ISSUE TITLE
class function replaced with enum.

### ISSUE THREAD

**Issue by destman** [nicklockwood/SwiftFormat#1195]
What rule cause such replacement?  Or maybe this is a bug?
``` diff
- class override func show() {
+ enum override func show() {
```

**Comment by nicklockwood** [comment]
That's the [enumNamespaces](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#enumNamespaces) rule

**Comment by destman** [comment]
Hm. Looks like bug for me. No new types was created here. Just override of class function...

**Comment by nicklockwood** [comment]
Oh sorry, I didn't look closely enough. Yes, this is clearly wrong 🤦‍♂️

**Comment by nicklockwood** [comment]
@destman can you provide the whole source file, or a cutdown standalone sample the reproduces the issue when swiftformat is run on it? I can't reproduce it from that one line.

**Comment by destman** [comment]
> @destman can you provide the whole source file, or a cutdown standalone sample the reproduces the issue when swiftformat is run on it? I can't reproduce it from that one line.

Shure. Here is minimal example that triggers this behaviour:
```swift
class Test {
    class override func show() {
        Boarding.instance?.present(instantinate())
    }
}

```
Version of swiftformat is 0.49.9.

**Comment by nicklockwood** [comment]
@destman fixed in 0.49.10
